src/sql/azdata.proposed.d.ts

```diff
@@ -1486,6 +1486,17 @@ declare module 'azdata' {
 			 */
 			viewModel: DesignerViewModel;
 		}
+
+		export interface PreviewReportResult {
```

Comment:
I was suggesting just naming this `PreviewReport`. After all - it's an object that is just describing the shape and format of the report so that makes sense to me.

Reply:
Following the convention for all other request types in table designer here. E.g. - `PublishChangesResult`, `DesignerEditResult` etc

Reply:
Ah. Ok, well not my favorite pattern for naming but I'll leave that up to you and @alanrenmsft to decide.